File: TugasPraktikum04/Bahasa Python/PRAK402-2310817320013-AikoAnatashaWendiono.py
input = int(input("Input: "))

for i in range(1, input + 1, 2):
    print(i, end=" ")

print()

if (input % 2 == 0):
    for i in range(input, 1, -2):
        print(i, end=" ")
elif (input % 2 == 1):
    for i in range(input - 1, 1, -2):
        print(i, end=" ")

File: TugasPraktikum04/Bahasa C/PRAK402-2310817320013-AikoAnatashaWendiono.c
#include<stdio.h>

int main(){

    int i, input;

    printf("Input: ");
    scanf("%d", &input);
    
    for (i = 1; i <= input; i = i + 2){
        printf("%d ", i);
    } printf("\n");
    if (input % 2 ==0){
        for (i = input; i >= 2; i = i - 2){
            printf("%d ", i);
        }
    }
    else if (input % 2 == 1){
        for (i = input - 1; i >= 2; i = i - 2){
            printf("%d ", i);
        }
    }
     return 0;
    
}
